src/ast/nodes/CallExpression.ts

```diff
@@ -1,14 +1,17 @@
 import CallOptions from '../CallOptions';
 import ExecutionPathOptions from '../ExecutionPathOptions';
 import SpreadElement from './SpreadElement';
-import { isGlobalVariable } from '../variables/GlobalVariable';
 import { isIdentifier } from './Identifier';
 import { ForEachReturnExpressionCallback, SomeReturnExpressionCallback } from './shared/Expression';
 import { isNamespaceVariable } from '../variables/NamespaceVariable';
 import { NodeType } from './NodeType';
-import { ExpressionNode, NodeBase } from './shared/Node';
+import { ExpressionNode, Node, NodeBase } from './shared/Node';
 import { ObjectPath } from '../values';
 
+export function isCallExpression(node: Node | { type?: string }): node is CallExpression {
```

Comment:
This seems to be defined but not actually used?

Reply:
Is removed👍